designs/waralee/src/mini.mjs

```diff
@@ -57,23 +61,23 @@ function waraleeMini({ options, Path, points, paths, complete, sa, macro, store,
 
     points.pText1 = points.mHip
       .shiftFractionTowards(points.mLeg, 0.5)
-      .attr('data-text', 'thisIsNotAPart')
+      .attr('data-text', 'waralee:thisIsNotAPart')
```

Comment:
Tip: we have a better API for this now: 
```js
points.addText('waralee:thisIsNotAPart', 'center')
``` 
See: https://freesewing.dev/reference/api/point/addtext